### PR TITLE
[SYCLomtaic-test]Enable cuda backend for features and help-function

### PR DESCRIPTION
**Pull request by wenhuiNi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ The format of the option map is key and value pairs. The key is the test option
 | "option_usmnone_cpu" : "--usm-level=none" | a. c2s migrates with usm switching off.  b. The test run on the CPU device. |
 | "option_gpu" : ""                         | a. c2s migrates with default opt. b. The test run on the GPU device. |
 | "option_usmnone_gpu" : "--usm-level=none" | a. c2s migrates with usm switching off. b. The test run on the GPU device.  |
+| "option_cuda" : ""                        | a. c2s migrates with default opt. b. The test run on the CPU device. |
```
Comment (ShengchenJ) [inline README.md]: Should on the NV backend.

File: option_mapping.json [hunk 2]
```diff
@@ -16,5 +16,6 @@
         "option_cpu_custom_helper_files_all": "--use-custom-helper=all",
         "option_usmnone_cpu_custom_helper_files_all": "--use-custom-helper=all --usm-level=none",
         "option_cpu_custom_helper_files_api": "--use-custom-helper=api",
-        "option_usmnone_cpu_custom_helper_files_api": "--use-custom-helper=api --usm-level=none"
+        "option_usmnone_cpu_custom_helper_files_api": "--use-custom-helper=api --usm-level=none",
+        "option_cuda" : ""
```
Comment (yihwang-nv) [inline option_mapping.json]: Add new line

File: README.md (hunk 1 above)
Comment (tangjj11) [inline README.md]: why CPU?

File: option_mapping.json (hunk 2 above)
Comment (tangjj11) [inline option_mapping.json]: redundant spaces before `:`

File: run_test.py [hunk 3]
```diff
@@ -479,11 +481,10 @@ def parse_input_args():
         exit(1)
     return args
 
-def main():
-    set_default_compiler()
+def main():
```
Comment (tangjj11) [inline run_test.py]: redundant spaces after `:`

**Review by tangjj11 — CHANGES_REQUESTED** [review]

**Review by ShengchenJ — CHANGES_REQUESTED** [review]

**Review by tangjj11 — APPROVED** [review]

**Review by ShengchenJ — APPROVED** [review]

**Pull request merged**